config/packages/test/webpack_encore.yaml

```diff
@@ -0,0 +1,2 @@
+#webpack_encore:
```

Comment:
This file should not exist. If needed, add test configs in main config file with `when@test`